Content.Client/GameObjects/Components/LowWallComponent.cs

```diff
@@ -33,6 +35,20 @@ public class LowWallComponent : IconSmoothComponent
         private IEntity _overlayEntity;
         private ISpriteComponent _overlaySprite;
 
+        [ViewVariables(VVAccess.ReadWrite)]
+        private string _overlayColor;
+
+        [ViewVariables(VVAccess.ReadWrite)]
+        private string _edgeColor;
+
+        public override void ExposeData(ObjectSerializer serializer)
+        {
+            base.ExposeData(serializer);
+
+            serializer.DataField(ref _overlayColor, "overlayColor", null);
```

Comment:
I *believe* the serializer will do to/from hex for you automatically if you give it a field which has type `Color`, so it should be possible to make your `string` fields `Color`s instead and cut out `FromHex` everywhere you've used it.

Reply:
Works